HW_4/task_2.py

```diff
@@ -0,0 +1,23 @@
+"""
+Задайте список случайных чисел. Выведите список чисел, которые не повторяются в заданном списке.
+
+Ввод: значение типа <list> (либо значения типа <int> – размерность списка)
+Вывод: значение типа <list>
+
+Пример:
+[1, 1, 2, 3, 3, 4, 5, 5, 6, 7, 7, 8, 9, 9]
+[2, 4, 6, 8]
+"""
+
+user_set = [int(i) for i in input('Введите последовательность чисел: ').split()]
```

Comment:
Пожалуйста, конкретизируйте.
```suggestion
user_set = [int(i) for i in input('Введите последовательность чисел через пробел: ').split()]
```